README.md

```diff
@@ -13,6 +13,13 @@ The Swedbank Pay SDK for PHP simplifies integrations against
 [Swedbank Pay's API Platform][api] by providing native PHP interface towards
 the REST API.
 
+This SDK include following payments options:
+
+* Credit and debit cards (Visa, Mastercard, Visa Electron, Maestro etc).
+* Invoice
+* Swish
+* Vipps
```

Comment:
Is Checkout supported as well?

Reply:
Yes checkout is supported. I think it is a little odd that Vipps is the only instrument that is excluded in the Checkout. Maybe it is a logical reason for this that I don't get?